### PR TITLE
Feature/flindt adheres to spindle styleguide

### PR DESCRIPTION
**Pull request by ghost**
Enjoy! :D
Mostly boring stuff with white lines / spaces and such, but have a critical look please.

### CODE REVIEW

**Review by hbokh — COMMENTED** [review]
LGTM

File: backend/flindt/user/views.py [hunk 1]
```diff
@@ -6,9 +6,8 @@
 from flindt.feedback.models import Feedback
 from flindt.feedback.serializers import FeedbackSerializer
 
-from .models import User, ExtraUserInfo, ExtraUserInfoCategory
-from .serializers import (UserSerializer, ExtraUserInfoSerializer,
-                          ExtraUserInfoCategorySerializer)
+from .models import ExtraUserInfo, ExtraUserInfoCategory, User
+from .serializers import (ExtraUserInfoCategorySerializer, ExtraUserInfoSerializer, UserSerializer,)
```
Comment (redmerloen) [inline backend/flindt/user/views.py]: I think the ( and ,) are redundant

File: backend/flindt/user/models.py [hunk 2]
```diff
@@ -60,28 +60,34 @@ class User(PermissionsMixin, AbstractBaseUser):
     objects = UserManager()
 
     def get_full_name(self):
-        return "%s" % self.first_name
+        return '%s' % self.first_name
```
Comment (redmerloen) [inline backend/flindt/user/models.py]: the string replacement is not necessary

File: backend/flindt/user/models.py [hunk 3]
```diff
@@ -60,28 +60,34 @@ class User(PermissionsMixin, AbstractBaseUser):
...
 
     def get_short_name(self):
-        return "%s %s" % (self.first_name, self.last_name)
+        return '%s %s' % (self.first_name, self.last_name)
```
Comment (redmerloen) [inline backend/flindt/user/models.py]: maybe replace this one with
```
return "{first_name} {last_name}".format(
first_name: self.first_name,
last_name: self.last_name,
)
```

**Comment by ghost** [comment]
Consistent string formatting and added how to contribute sections to the readme, as well as the cosmetic fixes from @spindleredmer

**Pull request merged**